platform/security/idp/security-idp-server/src/main/java/org/codice/ddf/security/idp/server/IdpEndpoint.java

```diff
@@ -1137,7 +1137,8 @@ public Response processPostLogout(
     try {
       if (samlRequest != null) {
         LogoutRequest logoutRequest =
-            logoutMessage.extractSamlLogoutRequest(RestSecurity.inflateBase64(samlRequest));
+            logoutMessage.extractSamlLogoutRequest(
```

Comment:
✏️ Ah. This was wrong to begin with. `DEFLATE` encoding is only appropriate for the Redirect binding, not POST and we should never have been trying to re-inflate the SAML request parameter. I would update `RestSecurity` moving this functionality to base64 decode there. Also, the `inflateBase64` method in `RestSecurity` could then be updated to rely on that for its internal decoding prior to re-inflation.